src/Common/src/TypeSystem/IL/TypeSystemContext.DelegateInfo.cs

```diff
@@ -34,18 +30,7 @@ protected override bool CompareValueToValue(DelegateInfo value1, DelegateInfo va
             }
             protected override DelegateInfo CreateValueFromKey(TypeDesc key)
             {
-                if (_supportLevel == CoreLibSupportLevel.Unknown)
-                {
-                    // Check if the core library supports dynamic invoke.
-                    _supportLevel = DelegateInfo.SupportsDynamicInvoke(key.Context) ?
```

Comment:
`DelegateInfo.SupportsDynamicInvoke` does not seem to be used anymore after this change.